### PR TITLE
Add TuneIn 1.19

### PR DESCRIPTION
**Pull request by boboldehampsink**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/tunein.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "tunein" do
+  version "1.19.0"
+  sha256 "591b77049d1e95e61335268fbaded1c6ace515d46b38ea8fb673ca873e3d16a5"
+
+  # amazonaws.com was verified as official when first introduced to the cask
+  url "https://s3-us-west-2.amazonaws.com/gemini-desktop-prod/release/TuneIn-#{version}.dmg"
```
[the remark this answers is not in the document]
Comment (boboldehampsink) [reply Casks/tunein.rb:6]: If you go to www.tunein.com - it is the big "Download App" link in the header menu

[the remark this answers is not in the document]
Comment (miccal) [reply Casks/tunein.rb:6]: Thanks @boboldehampsink, not sure how I missed that!

**Comment by miccal** [comment]
Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

**Comment by boboldehampsink** [comment]
@miccal done

**Comment by miccal** [comment]
Thank you @boboldehampsink.

**Review by miccal — APPROVED** [review]

**Pull request merged**